index.html

```diff
@@ -7705,7 +7730,7 @@ <h5>Presentational Role Inheritance</h5>
 						<th class="role-required-properties-head">Required States and Properties:</th>
 						<td class="role-required-properties">
 							<ul>
-								<li><pref>aria-valuenow</pref></li>
+								<li><pref>aria-valuenow</pref> or <pref>aria-valuetext</pref></li>
```

Comment:
I don't think we normally do ors here. The requirement is already specified in prose as a MUST so I think we should just remove the required property here